### PR TITLE
Cleanup contrib/backends/CDR

### PR DESCRIPTION
**Pull request by vincentschen**
* Run notebook + show outputs
* Add instructions for downloading auxiliary files

**Reminder to uncompress db**
```
bzip2 -d cdr.db.bz2
```

**Word embeddings:**
```
wget http://nlp.stanford.edu/data/glove.6B.zip \
&& mkdir -p glove.6B \
&& unzip glove.6B.zip -d glove.6B \
&& rm glove.6B.zip
```

### CODE REVIEW

File: metal/contrib/backends/CDR.ipynb [hunk 1]
```diff
@@ -20,22 +20,35 @@
...
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
     "import os\n",
     "import sys\n",
+    "sys.path.append('/dfs/scratch0/vschen/metal')\n",
```
Comment (ajratner) [inline metal/contrib/backends/CDR.ipynb]: Remove this

**Comment by ajratner** [comment]
@vincentschen you can merge after deleting that one line!

File: metal/contrib/backends/CDR.ipynb (hunk 1 above)
Comment (vincentschen) [reply metal/contrib/backends/CDR.ipynb]: Thanks for the catch!

**Comment by vincentschen** [comment]
> @vincentschen you can merge after deleting that one line!

Done!

**Pull request merged**